client/src/domain/event/EventHero.tsx

```diff
@@ -0,0 +1,285 @@
+    alert("TODO: Share event");
+  };
+
+  const isFree = React.useMemo(() => {
+    const price = eventData.linkedEventsEventDetails.offers.length
+      ? eventData.linkedEventsEventDetails.offers[0]
+      : null;
+
+    return !price || price.isFree;
+  }, [eventData.linkedEventsEventDetails.offers]);
+
+  const getPriceStr = () => {
+    return isFree
+      ? t("event.hero.offers.isFree")
+      : eventData.linkedEventsEventDetails.offers
+          .map(offer => getLocalisedString(offer.price || {}, locale))
+          .filter(e => e)
+          .sort()
+          .join(", ");
+  };
+
+  const image = eventData.linkedEventsEventDetails.images.length
+    ? eventData.linkedEventsEventDetails.images[0]
+    : null;
+  const description = eventData.linkedEventsEventDetails.shortDescription || {};
+  const keywords = eventData.linkedEventsEventDetails.keywords;
+  const startTime = eventData.linkedEventsEventDetails.startTime;
+  const endTime = eventData.linkedEventsEventDetails.endTime;
+  const name = eventData.linkedEventsEventDetails.name;
+  const today = isToday(new Date(startTime));
+  const thisWeek = isThisWeek(new Date(startTime));
+
+  return (
+    <div className={styles.heroWrapper}>
+      <Container>
+        <div className={styles.contentWrapper}>
+          <div className={styles.backButtonWrapper}>
+            <button className={styles.backButton} onClick={handleBack}>
+              <ArrowLeftIcon />
+            </button>
+          </div>
+          <div>
+            {image && (
+              <div
+                className={styles.image}
+                style={{ backgroundImage: `url(${image.url})` }}
+              />
+            )}
+          </div>
+          <div className={styles.leftPanel}>
+            <div className={styles.textWrapper}>
+              {(today || thisWeek || (!!keywords && !!keywords.length)) && (
+                <div className={styles.categoryWrapper}>
+                  {keywords &&
+                    keywords.map(keyword => {
+                      return (
+                        <Keyword
+                          key={keyword.id}
+                          keyword={getLocalisedString(keyword.name, locale)}
+                        />
+                      );
+                    })}
+                  {today && !thisWeek && (
+                    <Keyword color="lightEngel50" keyword="Today" />
+                  )}
+                  {!today && thisWeek && (
+                    <Keyword color="lightEngel50" keyword="This week" />
+                  )}
+                </div>
+              )}
+              <div className={classNames(styles.date, styles.desktopOnly)}>
+                {getDateStr(startTime, endTime)}
+              </div>
+              <div className={styles.title}>
+                {getLocalisedString(name, locale)}
+              </div>
+              <div
+                className={styles.description}
+                dangerouslySetInnerHTML={{
```

Comment:
Why does this need to be set dangerously?

Reply:
I forgot to change this. It used to have data that contains html. But not needed any more